registry/server/db.ts

```diff
@@ -8,7 +8,7 @@ const client: string = config.get('database.client');
 const knexConf: knex.Config = { // after: const knex = require('knex')({client: 'mysql'});
     client: client,
     connection: config.get('database.connection'),
-
+    useNullAsDefault: true,
```

Comment:
it looks strange. please add a comment on why did you set it.

Reply:
Added